lib/bundler/friendly_errors.rb

```diff
@@ -21,6 +21,10 @@ def self.with_friendly_errors
   rescue Thor::Error => e
     Bundler.ui.error e.message
     exit 1
+  rescue NameError => e
```

Comment:
this isn't necessarily true, there could be a bug in bundler that causes name errors